packages/core/src/data-grid/data-grid-types.ts

```diff
@@ -127,6 +127,7 @@ export interface GridMouseOutOfBoundsEventArgs extends BaseGridMouseEventArgs {
     readonly location: Item;
     readonly direction: readonly [-1 | 0 | 1, -1 | 0 | 1];
     readonly isMaybeScrollbar: boolean;
+    readonly innerDirection: readonly [-1 | 0 | 1, -1 | 0 | 1];
```

Comment:
I've been thinking about this and I just don't like the name. Do you have any other suggestions to help us find a better name?